src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md

```diff
@@ -116,6 +108,35 @@ browsers to be used to access your application.
 Ultimately, you should select the appropriate combination of queries to only
 target browsers that fit your needs.
 
+### @babel/preset-modules
+
+`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
+them based on the target browsers specified. Although this works well, an entire collection of
+syntax features is transformed when a targeted browser contains a bug with just a single feature.
+This results in more transformed code than is necessary.
+
+A separate preset, [`@babel/preset-modules`](https://github.com/babel/preset-modules) solves this
```

Comment:
If `bugfixes: true` and `preset-modules` are equivalent, should we even mention the latter? if they are equivalent today, will it remain that way or will the latter be more comprehensive?

Reply:
I took a swing at trying to reword to address this, let me know what you think @housseindjirdeh.